packages/shadydom/src/patches/Element.js

```diff
@@ -130,11 +130,12 @@ export const ElementPatches = utils.getOwnPropertyDescriptors({
     if (this.ownerDocument !== doc) {
       return this[utils.NATIVE_PREFIX + 'toggleAttribute'](attr, force);
     } else if (!scopeClassAttribute(this, attr, '')) {
-      this[utils.NATIVE_PREFIX + 'toggleAttribute'](attr, force);
+      const result = this[utils.NATIVE_PREFIX + 'toggleAttribute'](attr, force);
       distributeAttributeChange(this, attr);
+      return result;
     } else if (this.getAttribute(attr) === '' && !force) {
```

Comment:
Can you explain why the check for `!force` is here?

Reply:
Added explanation in comment and updated the comment in removeAttribute()